frontend/src/i18n/locales/features/landing-page/en-sg.ts

```diff
@@ -112,13 +112,9 @@ export const enSG: LandingPage = {
       },
       increaseAttachmentSizeLimit: {
         question: 'How do I increase attachment size limit?',
-        answer: `
-          The current size limit is 7 MB for email mode forms, and 20 MB for storage mode forms.
+        answer: `The current size limit 20 MB for storage mode forms.
```

Comment:
Typo: 
Wondering if there is a typo here: 
The current size limit `is` 20 MB for storage mode forms